### PR TITLE
feat: Improve React docs

### PR DESCRIPTION
**Pull request by dcramer**

### CODE REVIEW

File: src/collections/_documentation/platforms/javascript/getting-started-dsn/react.md [hunk 1]
```diff
@@ -1,42 +1,9 @@
-<!-- WIZARD -->
```
Comment (dcramer) [inline src/collections/_documentation/platforms/javascript/getting-started-dsn/react.md]: i think this was copy paste - afaict it shouldnt be needed

File: src/collections/_documentation/platforms/javascript/react.md [hunk 2]
```diff
@@ -4,17 +4,19 @@ sidebar_order: 30
 ---
 <!-- WIZARD -->
 To use Sentry with your React application, you will need to use `@sentry/browser` (Sentry’s browser JavaScript SDK).
```
Comment (dcramer) [inline src/collections/_documentation/platforms/javascript/react.md]: I would insert the install/configure instructions here, then error boundaries comes after they're fully setup

**Comment by dcramer** [comment]
jsx support doesnt seem to exist/work

image

image

**Comment by dcramer** [comment]
image

**Comment by dcramer** [comment]
Updated to remove version

File: src/collections/_documentation/platforms/javascript/getting-started-dsn/react.md (hunk 1 above)
Comment (cameronmcefee) [reply src/collections/_documentation/platforms/javascript/getting-started-dsn/react.md]: Confirmed, this doc doesn't appear to be referenced in platforms.yml

**Comment by dcramer** [comment]
Jk my jsx is wrong

**Comment by dcramer** [comment]
To me this is an improvement if I stumble upon the React page itself, but something we should decide broadly: do we embed the setup docs on every framework, or should we just have a generic entry blurb that directs them to the core JavaScript setup. In some framework cases the setup will change so it can still make sense.

**Pull request merged**